src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const firstState = { ...state };
+  const changeState = [];
+
+  function assigned(index, to, from) {
+    changeState[index] = Object.assign(to, from);
+  }
```

Comment:
This function is redundant, you can use destructuring for creating object copy and add it to array
array.push(...object)